### ISSUE TITLE
can't run make

### ISSUE THREAD

**Issue by iustinian91** [cockpit-project/cockpit-files#763]
I have followed the installation instructions and when I run `make` I get this error: 
`make: *** [Makefile:48: pkg/lib/cockpit-po-plugin.js] Error 129`

any suggestions on what I might be doing wrong?

**Comment by jelly** [comment]
Do you have a few more lines of output, the distro you use and architecture?

**Comment by iustinian91** [comment]
Yes! I'm running this under Ubuntu 20.04, x64. Here is the entire output:
`root@server:~/cockpit-files# make
error: unknown option `no-write-fetch-head'
usage: git fetch [<options>] [<repository> [<refspec>...]]
   or: git fetch [<options>] <group>
   or: git fetch --multiple [<options>] [(<repository> | <group>)...]
   or: git fetch --all [<options>]

    -v, --verbose         be more verbose
    -q, --quiet           be more quiet
    --all                 fetch from all remotes
    --set-upstream        set upstream for git pull/fetch
    -a, --append          append to .git/FETCH_HEAD instead of overwriting
    --upload-pack <path>  path to upload pack on remote end
    -f, --force           force overwrite of local reference
    -m, --multiple        fetch from multiple remotes
    -t, --tags            fetch all tags and associated objects
    -n                    do not fetch all tags (--no-tags)
    -j, --jobs <n>        number of submodules fetched in parallel
    -p, --prune           prune remote-tracking branches no longer on remote
    -P, --prune-tags      prune local tags no longer on remote and clobber changed tags
    --recurse-submodules[=<on-demand>]
                          control recursive fetching of submodules
    --dry-run             dry run
    -k, --keep            keep downloaded pack
    -u, --update-head-ok  allow updating of HEAD ref
    --progress            force progress reporting
    --depth <depth>       deepen history of shallow clone
    --shallow-since <time>
                          deepen history of shallow repository based on time
    --shallow-exclude <revision>
                          deepen history of shallow clone, excluding rev
    --deepen <n>          deepen history of shallow clone
    --unshallow           convert to a complete repository
    --update-shallow      accept refs that update .git/shallow
    --refmap <refmap>     specify fetch refmap
    -o, --server-option <server-specific>
                          option to transmit
    -4, --ipv4            use IPv4 addresses only
    -6, --ipv6            use IPv6 addresses only
    --negotiation-tip <revision>
                          report that we have only objects reachable from this object
    --filter <args>       object filtering
    --auto-gc             run 'gc --auto' after fetching
    --show-forced-updates
                          check for forced-updates on all updated branches
    --write-commit-graph  write the commit-graph after fetching

make: *** [Makefile:48: pkg/lib/cockpit-po-plugin.js] Error 129`

**Comment by iustinian91** [comment]
I have made some progress. I updated `nvm` from version 10 to version 18.

 I also replaced the this like in the _Makefile_
`git fetch --no-tags --no-write-fetch-head --depth=1 $(COCKPIT_REPO_URL) $(COCKPIT_REPO_COMMIT)`
with
`git fetch --no-tags --depth=1 $(COCKPIT_REPO_URL) $(COCKPIT_REPO_COMMIT)`

Removing `--no-write-fetch-head` did the trick and finished with `15:58:36: Build finished in 1113 ms`.

Now after I've run `make install` and restarted Cockpit I can't see the _Cockpit Files_ in the sidebar. I have checked `/root/.local/share/cockpit/cockpit-files` and has the files in it.
